docs/tests.md

```diff
@@ -1,16 +1,15 @@
 ## Test
 
-- Run local tests against source code. </br> `poe test_src` </br> </br> 
-  - Profiles: </br>
-    - Default: Runs only mypy and pyright tests </br> `poe test_src --profile=default` </br>
-    - Pytest: Runs only pytest </br> `poe test_src --profile=pytest` </br> 
-    - Pre-commit: Runs all style checks in pre-commit </br> `poe test_src --profile=style` </br> 
-    - Full: Run all the above tests </br> `poe test_src --profile=full` </br> </br>
+[Poe](https://github.com/nat-n/poethepoet) is used to run all tests.
 
-- Run local tests against distribution: </br> `poe test_dist` </br> </br>
+Here are the most important options. Fore more details, please use `poe --help`.
 
-- Run all local tests: </br> `poe test_all` </br> </br>
+- Run all tests (includes installing the stubs): `poe test_all`
```

Comment:
I think this should say "includes testing the stubs when installed"

Reply:
`test_all` installes the stubs, then tests them, and then uninstalled them

Reply:
Sure, but the remark seems to indicate that it is _just_ installing the stubs.  So that's why I suggested the change.

How about "includes testing stubs as installed"

Reply:
I merged main and slightly modified the existing description